### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by timtroendle**

### CODE REVIEW

File: web/app/configs/lang.json [hunk 1]
```diff
@@ -376,5 +376,68 @@
...
+    "hydro": "Wasser",
+    "hydro storage": "Pumpspeicher",
+    "biomass": "Biomasse",
+    "nuclear": "Atomkraft",
+    "geothermal": "Geothermie",
+    "gas": "Gas",
+    "coal": "Kohle",
+    "oil": "Öl",
+    "unknown": "unbekannt",
+    "exportto": "des Exports zu",
```
Comment (timtroendle) [inline web/app/configs/lang.json]: What's the full sentence here?

File: web/app/configs/lang.json [hunk 2]
```diff
@@ -376,5 +376,68 @@
...
+    "importfrom": "des Imports von",
+    "productionimport": "Produktion/Import",
+    "consumption": "Verbrauch",
+    "electricityfrom": "Elektrizität kommt von",
+    "electricityto": "Elektrizität wird exportiert nach",
+    "electricitystored": "Elektrizität wird gespeichert mit",
```
Comment (timtroendle) [inline web/app/configs/lang.json]: What's the full sentence here?

File: web/locales/de.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        "back": "zurück",
+        "source": "Quelle",
+        "fossilfuels": "fossile Brennstoffe",
+        "electricityprice": "Elektrizitätspreis (day-ahead)",
+        "electricityproduction": "Elektrizitätsproduktion",
+        "showemissions": "zeige Emissionen",
+        "bysource": "nach Quelle",
+        "emissions": "Emissionen",
+        "showelectricity": "zeige Elektrizität",
+        "co2signal": "Unser <a href=\"http://www.co2signal.com\" target=\"blank\">CO2 Signal</a> hilft Geräten und elektrischen Fahrzeugen Elektrizität zur richtigen Zeit zu konsumieren, insbesondere bei Grenzübertritt."
```
Comment (timtroendle) [inline web/locales/de.json]: Does that literally mean: it helps especially when you are travelling across borders? That's the way I translated it at least.

File: web/locales/de.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    "tooltips": {
+        "electricityprice": "Elektrizitätspreis (day-ahead)",
+        "fossilfuels": "fossile Brennstoffe",
+        "crossborderexport": "Export über Ländergrenzen",
+        "carbonintensityexport": "CO2 Konzentration des Exports",
+        "of": "von",
+        "ofconsumption": "des",
+        "ofinstalled": "der installierten Leistung",
+        "ofelectricity": "der",
+        "utilizing": "verwendend",
```
Comment (timtroendle) [inline web/locales/de.json]: What's the full sentence here?

File: web/app/configs/lang.json (hunk 1 above)
Comment (ThierryOllivero) [inline web/app/configs/lang.json]: unbekannt ist ok

For "Oil", it's to be understood as "Erdöl" I believe. But maybe "Öl" is what German use ?

For Export to, I'm not sure it is still used, I did not find it in the map.  But your translation seems ok.

File: web/app/configs/lang.json (hunk 2 above)
Comment (ThierryOllivero) [inline web/app/configs/lang.json]: 10% of FR electricity comes from DE
10% of  FR electricity is exported to DE
10% of FR electricity is stored to hydro storage

So for your translation, I don't know if "wird" is better than "ist" ? You know better :)

File: web/locales/de.json (hunk 3 above)
Comment (ThierryOllivero) [inline web/locales/de.json]: Yes, that's what it means.

File: web/locales/de.json (hunk 4 above)
Comment (ThierryOllivero) [inline web/locales/de.json]: 67% of FR electricity comes from nuclear
utilizing 81% of installing capacity

File: web/app/configs/lang.json (hunk 1 above)
Comment (timtroendle) [inline web/app/configs/lang.json]: True `Erdöl` is more precise, same for gas. Changed it.

File: web/app/configs/lang.json (hunk 2 above)
Comment (timtroendle) [inline web/app/configs/lang.json]: `Ist` works only in the past tense here as in `ist exportiert worden`, `wird exportiert` is present tense. ;-)

File: web/locales/de.json (hunk 4 above)
Comment (timtroendle) [inline web/locales/de.json]: Ok, so `durch Verwendung von` should do it.

File: web/app/configs/lang.json (hunk 2 above)
Comment (ThierryOllivero) [inline web/app/configs/lang.json]: OK thanks :)

**Comment by corradio** [comment]
@ThierryOllivero do you approve?

**Comment by ThierryOllivero** [comment]
Yes, worst case scénario our friend will help us correct the mistakes.

**Comment by corradio** [comment]
ok then you should approve on github :) Merging anyway. Thanks for the help @timtroendle !

**Pull request merged**